api/proto/common/cron_pub.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
```

Comment:
The proto file will also have circular dependency detection. The cron proto references the PipelineCreateRequest structure of pipeline, so it can only be placed in common